README.md

```diff
@@ -34,28 +36,31 @@ Flush an image
 
 ## Usage as library
 
-#### 1. Add to composer
+#### 1. Add to composer dependencies
 
-`composer require badges/poser ~1.2`
+`$ composer require badges/poser ~1.2`
```

Comment:
I would avoid to suggest a version: composer is able to pick the best one.

Reply:
I agree! Above I had removed it! :D